snykTask/src/__tests__/test-task-args.ts

```diff
@@ -44,6 +44,23 @@ test('ensure no problems if both targetFile and docker-file-path are both not se
   expect(fileArg).toBe('');
 });
 
+test('ensure no problems if testType set to code and both targetFile and dockerImageName are both set', () => {
+  const args = defaultTaskArgs();
+  args.testType = TestType.CODE;
+  args.dockerImageName = 'some-docker-image';
+  args.targetFile = 'some-target-file';
+  args.dockerfilePath = null as any;
+
+  const fileArg = args.getFileParameter();
+  console.log(`fileArg: ${fileArg}`);
+  if (fileArg == null) {
+    console.log('fileArg is null');
+  }
+
+  expect(args.testType).toBe('code');
+  expect(fileArg).toBe('');
+});
+
```

Comment:
I wonder if we could rephrase this unit test message to something easier to understand when a reviewer doesn't have the context of the change.

If I understood it correctly, we could change the message to `"test that fileArg is empty when testType is 'code' with both targetFile and dockerImageName set"`. This way, anyone reading the test can understand its purpose at a glance.

Reply:
Maybe we could reprase both test that start with "ensure no problem", since it's sounds a bit vague? If I understand correctly what they are doing I'd say:

```ts
test('fileArg should be an empty string when both targetFile and dockerfilePath are not set', () => {...});

test('fileArg should be an empty string when testType is set to CODE and both targetFile and dockerImageName are provided', () => {});
```

Reply:
yes, explicit definition of its test output state is better. This is now revised.